twill11/hometask-6/lib/csv_writer.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'csv'
+require_relative 'parser'
+
+# Writing info to csv file
+class CsvWriter
+  HEADERS = ['photo id', 'person real name', 'photo title', 'link to image'].freeze
+
+  attr_reader :data
+
+  def initialize(data)
+    @data = data
+  end
+
+  def csv_writer(tag)
+    CSV.open("file/#{tag}.csv", 'wb') do |csv|
+      csv << HEADERS
```

Comment:
I don't have `file`  folder and see the error.

`/home/vital/.rvm/rubies/ruby-3.0.2/lib/ruby/3.0.0/csv.rb:1421:in `initialize': No such file or directory @ rb_sysopen - file/dog.csv (Errno::ENOENT)`